drivers/wifi/eswifi/Kconfig.eswifi

```diff
@@ -30,6 +30,13 @@ config WIFI_ESWIFI_BUS_UART
 
```

Comment:
Please change the commit subject to `drivers: eswifi: ...`

Reply:
Done

Reply:
I may have gotten a little carried away with the sync fork, I'll look into fixing that.